common/lib/xmodule/xmodule/js/src/video/07_video_volume_control.js

```diff
@@ -39,12 +39,24 @@ function() {
 
         template: [
             '<div class="volume">',
-                '<a href="#" role="button" aria-disabled="false" title="',
-                    gettext('Volume'), '" aria-label="',
-                    gettext('Click on this button to mute or unmute this video or press UP or DOWN buttons to increase or decrease volume level.'),
-                    '"></a>',
-                '<div role="presentation" class="volume-slider-container">',
-                    '<div class="volume-slider"></div>',
+                '<button class="control" aria-disabled="false" aria-label="',
+                    gettext('Volume: Click on this button to mute or unmute this video or press UP or ',
```

Comment:
I've seen reports that ``gettext`` doesn't get scraped correctly when it is broken across lines, so this may not work when translated. I recommend keeping it all on the same line.

In fact, I don't think this line is using the correct multi-line syntax. You should be using ``+`` but in fact you've used ``,`` which will make this into two separate arguments to gettext, the second of which will be ignored.